pkg/cases/handler.go

```diff
@@ -8,14 +8,106 @@ import (
 type CaseHandler struct {
 }
 
-// @Summary Get case by case id
+// @Summary New a case
 // @Description
+// @Tags Case
 // @Produce json
-// @Param  id path uint true "case ID"
+// @Param caseDetail body string true "patient ID, doctor ID and department name"
 // @Success 200 {object} api.ReturnedData{data=Case}
-// @Router /case/{id} [GET]
-func (h *CaseHandler) GetCaseByCaseID(c echo.Context) error {
+// @Router /case [POST]
+func (h *CaseHandler) NewCase(c echo.Context) error {
+	// ...
+	c.Logger().Debug("hello world")
+	return c.JSON(200, api.Return("NewCase", nil))
+}
+
+// @Summary Get case by a case-id list
```

Comment:
You so where would front end get this case-id list?
It should be get a list of cases from one id